### PR TITLE
Auto bootstrap failling on non seed nodes

### PR DESCRIPTION
**Pull request by chicoandrade**
Im getting the message bellow when starting a non seed node:
- This node will not auto bootstrap because it is configured to be a seed node.

Looking into the yaml ive saw that alongside the right seed ips was the ip of the node i was starting.
Ive found a suspicious line at the docker-entrypoint.sh:
{code}
	if [ "$CASSANDRA_SEEDS" ]; then
		CASSANDRA_SEEDS="$CASSANDRA_SEEDS,$CASSANDRA_BROADCAST_ADDRESS"
	else
		CASSANDRA_SEEDS="$CASSANDRA_BROADCAST_ADDRESS"
	fi
{code}

### CODE REVIEW

**Comment by yosifkit** [comment]
Thanks for the fix. Wouldn't it be a slightly better to just drop the whole `if` and replace it with a conditional assignment? :wink:
```bash
: ${CASSANDRA_SEEDS="$CASSANDRA_BROADCAST_ADDRESS"}
```
Can we also get this in the `docker-entrypoint.sh` at the root of the repo?

**Comment by chicoandrade** [comment]
Sure seems a better solution :)

**Comment by yosifkit** [comment]
LGTM, ping @tianon

File: 2.0/docker-entrypoint.sh [hunk 1]
```diff
@@ -20,11 +20,8 @@ if [ "$1" = 'cassandra' ]; then
 	fi
 	: ${CASSANDRA_BROADCAST_RPC_ADDRESS:=$CASSANDRA_BROADCAST_ADDRESS}
 
-	if [ "$CASSANDRA_SEEDS" ]; then
-		CASSANDRA_SEEDS="$CASSANDRA_SEEDS,$CASSANDRA_BROADCAST_ADDRESS"
-	else
-		CASSANDRA_SEEDS="$CASSANDRA_BROADCAST_ADDRESS"
-	fi
+	: ${CASSANDRA_SEEDS="$CASSANDRA_BROADCAST_ADDRESS"}
```
Comment (tianon) [inline 2.0/docker-entrypoint.sh]: This needs to be `:=`, not just `=`. :wink:

**Comment by chicoandrade** [comment]
Hi @tianon , I think thats it. Any more suggestions for the pull request?

**Comment by tianon** [comment]
LGTM

**Pull request merged**